src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -196,7 +195,7 @@ public object GetProperyValue(object obj, string propertyName)
                 var completed = await CompleteValue(context, context.Schema.FindType(nonNullType.Type), fields, result);
                 if (completed == null)
                 {
-                    throw new ExecutionError("Cannot return null for non-null type. Field: {0}".ToFormat(nonNullType.Name));
+                    throw new ExecutionError("Cannot return null for non-null type.");
```

Comment:
Yep, agree. Just most times, the name was null when I tested it, so didn't provide context. Will re-add.